tests/resource/valid/class/assign_types_double_nested.mamba

```diff
@@ -0,0 +1,19 @@
+class Y(def a: Float)
+
+class X
+    def y: Y
```

Comment:
Should be a test as well to check that all class variables are actually signed to in the constructor.
If not, then the type should be nullable.